packages/wrangler/src/metrics/helpers.ts

```diff
@@ -0,0 +1,57 @@
+import os from "node:os";
+import whichPmRuns from "which-pm-runs";
+import { version as wranglerVersion } from "../../package.json";
+
+export function getWranglerVersion() {
+	return wranglerVersion;
+}
+
+export function getPackageManager() {
```

Comment:
We already have logic for this in `src/package-manager.ts`—was there a reason to introduce a new dependency for this?

Reply:
argh! didn't see that - yeah i'll use that instead 👍

Reply:
This helper seems like a 1:1 passthrough?

Reply:
5fcef96670a2fa2e897f868dbce666a7efbe7b61